packages/web-components/src/components/va-alert/test/va-alert.e2e.ts

```diff
@@ -23,6 +23,9 @@ describe('va-alert', () => {
           </div>
         </mock:shadow-root>
         <h4 slot="headline">
+          <span class="usa-sr-only">
+            Information Alert 
+          </span>
           This is an alert
```

Comment:
We may cause test failures in vets-website if tests are checking for an exact match of the header text. I found a couple that are using `contain` though which should still pass. 

<img width="662" height="92" alt="Image" src="https://github.com/user-attachments/assets/1c789ba4-5ecb-4398-b4ff-e99fc8530e81" />